raiden/ui/console.py

```diff
@@ -216,27 +224,6 @@ def register_token(self, token_address_hex):
         self._raiden.register_channel_manager(channel_manager.address)
         return channel_manager
 
-    def ping(self, peer_address_hex, timeout=4):
```

Comment:
My proposal would be that `nonce=0` is reserved for user pings. In other words special handling of ping messages with `nonce=0`.